app/controllers/bikes_controller.rb

```diff
@@ -12,7 +12,6 @@ def new
   def create
     @bike = Bike.new(bike_params)
     @bike.user = current_user
-    @bike.save
```

Comment:
@mdpolbrich why did you delete the saving? The bike.new doesn´t save automatically. like the bike.upadate does. 
I think it has to go back in.